### ISSUE TITLE
Unable to Read Code 39 (0.16.4)

### ISSUE THREAD

**Issue by ALEXTHENINJA007** [micjahn/ZXing.Net#122]
Is there an issue with the current version on NuGet for Code 39 Barcodes? (version 0.16.4)

I can get QR and 128 fine, but not 39 (anything to do with BarcodeReader.Options in my code was just debuging basically), I tried various barcodes from what I had on hand and google images but no luck. Code 39 was the one I wanted, but I should be fine using 128 so just letting you know.

Code for decoding
```C#
//toScan is a public Bitmap created elsewhere;
BarcodeReader Reader = new BarcodeReader();
Reader.Options.PossibleFormats = new List<BarcodeFormat>()
     {
             BarcodeFormat.CODE_39,
             BarcodeFormat.QR_CODE,
              BarcodeFormat.CODE_128,
      };
  Reader.Options.PureBarcode = false;
 Reader.Options.TryHarder = true;
 Reader.Options.UseCode39RelaxedExtendedMode = true;
Result result = Reader.Decode(toScan);
```

Thanks,
Alex

**Comment by micjahn** [comment]
Do you have a sample image which works with pre-0.16.4?

**Comment by ALEXTHENINJA007** [comment]
No, I didn't start my project before pre-0.16.4

**Comment by micjahn** [comment]
Which .Net framework version are you using (targeting classic .Net, .Net Core or something else)? 
Can you provide a sample image which doesn't work for you?
All of my test images work as expected.

**Comment by ALEXTHENINJA007** [comment]
All good, I figured out my problem. 
I was using other packages to access the webcam to then use that image to scan, however I wasn't updating the bitmap correctly so it only got the first frame of the webcam, but my video feed in my application was updating correctly, took me ages to figure out. 
(Anything I scanned from my computer screen involved me picking up the device so the first frame basically never was correct. When scanning printed ones I'd always start with type 39 and get the webcam positioned perfectly and think surely this has to scan and then close the program check the logs and then I'd place down a different bar-code and start the app and it would work because the first frame was positioned perfectly  )
